src/Microsoft.ML.TimeSeries/PolynomialModel/LinearModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.TimeSeries
+{
+    /// <summary>
+    /// This class calculates f(x) = b0 + b1 * x
+    /// </summary>
+    internal sealed class LinearModel : AbstractPolynomialModel
+    {
+        /// <summary>
+        /// Store the coefficients in member variables for better performance.
+        /// </summary>
+        private readonly double _b0;
+        private readonly double _b1;
+
+        public LinearModel(IReadOnlyList<double> coeffs)
+            : base(coeffs)
+        {
+            Contracts.CheckParam(coeffs.Count == 2, nameof(coeffs), "must contain exact 2 elements.");
+
```

Comment:
Nit: "Must contain exactly 2 elements".

Reply:
Updated, thanks.

---
In reply to: [442497881](https://github.com/dotnet/machinelearning/pull/5202#discussion_r442497881) [](ancestors = 442497881)